megatron/training.py

```diff
@@ -623,8 +650,8 @@ def training_log(loss_dict, total_loss_dict, learning_rate, iteration,
             iteration, args.train_iters)
         log_string += ' consumed samples: {:12d} |'.format(
             args.consumed_train_samples)
-        log_string += ' elapsed time per iteration (ms): {:.1f} |'.format(
-            elapsed_time_per_iteration * 1000.0)
+        log_string += ' elapsed time per iteration (s): {:.1f} |'.format(
```

Comment:
Maybe it makes sense to increase the number of decimal numbers here to include the extra precision that was lost in the unit conversion. I.e., use `{:.4f}`, for example